spec/javascript/components/applications/application_form/__snapshots__/PaperApplicationForm.test.js.snap

```diff
@@ -0,0 +1,7290 @@
+                          "mailing_city": "SAN FRANCISCO",
+                          "mailing_state": "CA",
+                          "mailing_street": "123 MAIN ST",
+                          "mailing_zip_code": "94105-1804",
+                          "middle_name": null,
+                          "name": "karen jones",
+                          "phone": "9548883321",
+                          "phone_type": "Cell",
+                          "residence_address": "123 MAIN ST, SAN FRANCISCO, CA, 94105-1804",
+                          "second_phone": null,
+                          "second_phone_type": null,
+                          "state": "CA",
+                          "street": "123 MAIN ST",
+                          "zip_code": "94105-1804",
+                        },
+                        "application_language": "English",
+                        "application_submission_type": "Electronic",
+                        "application_submitted_date": "2018-06-19",
+                        "createdby": Object {
+                          "name": "Prod Vertiba",
+                        },
+                        "demographics": Object {},
+                        "flagged_applications": Array [],
+                        "has_ada_priorities_selected": Object {
+                          "hearing_impaired": true,
+                          "mobility_impaired": true,
+                          "vision_impaired": true,
+                        },
+                        "has_developmental_disability": null,
+                        "has_military_service": null,
+                        "household_members": Array [
+                          Object {
+                            "city": "buenos aires",
+                            "date_of_birth": "1976-06-11",
+                            "first_name": "diego",
+                            "last_name": "maradona",
+                            "middle_name": "m",
+                            "name": "diego m maradona",
+                            "relationship_to_applicant": null,
+                            "residence_address": "9 de julio, buenos aires, fl, 33026",
+                            "state": "fl",
+                            "street": "9 de julio",
+                            "zip_code": "33026",
+                          },
+                        ],
+                        "housing_voucher_or_subsidy": "true",
+                        "id": "a0o0x000000OcOzAAK",
+                        "listing": Object {
+                          "id": "a0W0x000000GhJUEA0",
+                          "name": "Test 5/30",
+                        },
+                        "lottery_number": "00191270",
+                        "lottery_number_manual": null,
+                        "monthly_income": null,
+                        "name": "APP-00191270",
+                        "preferences": Arra
```

Comment:
this error class on the form group is one thing we expect to see when the Annual Income field is in an error state